doc/dev/release.md

```diff
@@ -349,3 +370,9 @@ Add the following line to the top of the GitHub release notes for `v1.3.0`:
 [homebrew-readme]:https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#to-submit-a-version-upgrade-for-the-foo-formula
 [homebrew-repo]:https://github.com/Homebrew/homebrew-core
 [sdk-samples-repo]:https://github.com/operator-framework/operator-sdk-samples
+[upgrade-version-doc]:../migration/version-upgrade-guide.md
+[ansible-guide]:../ansible/user-guide.md
+[helm-guide]:../helm/user-guide.md
+[ansible-memcached-sample]:https://github.com/operator-framework/operator-sdk-samples/tree/master/memcached-operator
+[helm-memcached-sample]:https://github.com/operator-framework/operator-sdk-samples/tree/master/helm/memcached-operator
+[go-memcached-sample]:https://github.com/operator-framework/operator-sdk-samples/tree/master/memcached-operator
```

Comment:
What about the other operator examples? If they're present, I think we should keep maintaining them. 

@hasbro17 @estroz @fabianvf Should we keep the other sample operators around?

Also, should we move the Go memcached-operator into a `go` subdirectory to match the other project type locations?

Reply:
Hi @joelanford, 

We have been unable to keep maintained the Memcached ones. IHMO we should not have many samples and we should have just the Memcached ones and then, we should actually improve them to be a very good sample by such as adding tests, coveralls, ci and etc . I added this topic for we discussed in the tech meeting.

Reply:
I agree @camilamacedo86. Interested in what others think :wink: